lib/npg_qc/Schema/Result/ErrorsByCycle.pm

```diff
@@ -184,8 +184,11 @@ __PACKAGE__->belongs_to(
 );
 
 
-# Created by DBIx::Class::Schema::Loader v0.07036 @ 2015-06-30 16:51:56
-# DO NOT MODIFY THIS OR ANYTHING ABOVE! md5sum:EQTAxzV9URocOuAP4aF4CQ
+# Created by DBIx::Class::Schema::Loader v0.07047 @ 2017-12-01 15:01:58
```

Comment:
Maybe we can checkout the previous version of the file in devel so we don't need to regenerate it. Something like:

```bash
git fetch upstream devel
git checkout upstream/devel -- lib/npg_qc/Schema/Result/ErrorsByCycle.pm
git checkout upstream/devel -- lib/npg_qc/Schema/Result/MoveZ.pm
```

Then I re-ran schema loader and came out with no changes...